src/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -426,13 +426,9 @@ public void Count()
             Assert.Throws<InvalidOperationException>(() => ((IReadOnlyCollection<int>)s_emptyDefault).Count);
 
             Assert.Equal(0, s_empty.Length);
-            Assert.Equal(0, ((ICollection)s_empty).Count);
-            Assert.Equal(0, ((ICollection<int>)s_empty).Count);
             Assert.Equal(0, ((IReadOnlyCollection<int>)s_empty).Count);
 
             Assert.Equal(1, s_oneElement.Length);
-            Assert.Equal(1, ((ICollection)s_oneElement).Count);
-            Assert.Equal(1, ((ICollection<int>)s_oneElement).Count);
```

Comment:
We already test for count in various ICollection tests (see https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs and https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Collections/ICollection.Generic.Tests.cs)